### PR TITLE
Made a Node class

### PR DESCRIPTION
**Pull request by sylvestor88**
So this is what I perceived from Ariel's comment about implementing Node as object.
Let me know if I got it right or changes need to be made.

### CODE REVIEW

**Comment by ptnapoleon** [comment]
Yes, the code is fine, and you have the general idea, but I don't like putting this in the modules package, as it isn't a module. I'll also make some inline comments about the API.

File: test/validation/org/apache/cassandra/modules/Node.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.modules;
+
+public class Node
+{
+    private String node;
```
Comment (ptnapoleon) [inline test/validation/org/apache/cassandra/modules/Node.java]: I think this should be `name`, not `node`.

File: test/validation/org/apache/cassandra/modules/Node.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    {
+        this.node = node;
+    }
+
+    public String getNode()
+    {
+        return node;
+    }
+
+    public void setNode(String node)
```
Comment (ptnapoleon) [inline test/validation/org/apache/cassandra/modules/Node.java]: Then subsequently, these should be `getName` and `setName`

Comment (sylvestor88) [inline test/validation/org/apache/cassandra/modules/Node.java]: Will change

File: test/validation/org/apache/cassandra/modules/Node.java (hunk 1 above)
Comment (sylvestor88) [inline test/validation/org/apache/cassandra/modules/Node.java]: Will change

**Comment by sylvestor88** [comment]
Well I wasn't sure where to put the Node class so added to the Module package as for now modules will be calling it to instantiate a node. However, the other place I can think of is the bridge package. Would that be fine?

**Comment by ptnapoleon** [comment]
Don't put it in any package, just leave it in the root dir for now. When I can think of a good package name, I'll move it there

**Pull request merged**